RELEASENOTES.md

```diff
@@ -3,4 +3,5 @@
 - Renamed the CLI from octoshift to ado2gh to indicate that this one is specifically for Azure DevOps -> GitHub migrations (in the future there will be additional CLI's for other migration scenarios)
 - Released gei.exe that adds support for Github -> Github migrations (GHEC only for now). In the future this will be exposed as an extension to the Github CLI.
 - Automatically remove secrets from log files and console output (previously the verbose logs would contain your PAT's)
-- Added --ssh option to generate-script and migrate-repo commands (in both ado2gh and gei). This forces the migration to use an older version of the API's that uses SSH to push the repos into GitHub. The newer API's use HTTPS instead. However some customers have been running into problems with some repos that work fine using the older SSH API's. In the future this option will be deprecated once the issues with the HTTPS-based API's are resolved.
\ No newline at end of file
+- Added --ssh option to generate-script and migrate-repo commands (in both ado2gh and gei). This forces the migration to use an older version of the API's that uses SSH to push the repos into GitHub. The newer API's use HTTPS instead. However some customers have been running into problems with some repos that work fine using the older SSH API's. In the future this option will be deprecated once the issues with the HTTPS-based API's are resolved.
+- gei.exe can now optionally use GH_SOURCE_PAT for source.
```

Comment:
I don't think we need this bullet in the release notes because this is the first release with gei so addin the source pat is not a "change" it's just the way it is the first time the customers see it.

I WOULD however maybe add a blurb to the Description field in the two gei commands (and possibly all other commands) noting which env vars that specific command expects to be set.

Reply:
addressed in d9a01643f5ca0392832eb66d288995af18a3bfff